### ISSUE TITLE
bug: unable to apply patch from RVX Manager

### ISSUE THREAD

**Issue by hoang-himself** [inotia00/ReVanced_Extended#1091]
### Type

Error while patching

### Tools used

ReVanced Manager

### Application

YouTube v18.25.39

### Bug description

Error while patching

### Steps to reproduce

1. Use RVX Manager 1.3.8
2. Patch selection:
```json
"com.google.android.youtube": [
    "enable-minimized-playback",
    "enable-open-links-directly",
    "enable-seekbar-tapping",
    "force-opus-codec",
    "force-premium-heading",
    "force-vp9-codec",
    "hide-auto-player-popup-panels",
    "hide-cast-button",
    "hide-channel-watermark",
    "hide-crowdfunding-box",
    "hide-floating-microphone",
    "hide-general-ads",
    "hide-music-button",
    "hide-video-ads",
    "microg-support",
    "optimize-resource",
    "overlay-buttons",
    "return-youtube-dislike",
    "sponsorblock"
  ]
```

### Relevant log output

```shell
Initializing installer
Creating working directory
Copying original apk
Unpacking input apk
Reading dex files
Decoding AndroidManifest.xml only, because resources are not needed
Merging integrations
Deleting existing resource cache directory
Decoding resources
Executing patches
Failed to apply enable-minimized-playback: 'enable-minimized-playback' depends on 'player-type-hook' but the following error was raised: Failed to resolve YouTubeControlsOverlayFingerprint
Applied enable-open-links-directly
Applied enable-seekbar-tapping
Applied force-opus-codec
Applied force-premium-heading
Applied force-vp9-codec
Applied hide-auto-player-popup-panels
Applied hide-cast-button
Applied hide-channel-watermark
Applied hide-crowdfunding-box
Applied hide-floating-microphone
Failed to apply hide-general-ads: 'hide-general-ads' depends on 'LithoFilterPatch' but the following error was raised: 'LithoFilterPatch' depends on 'player-type-hook' but the following error was raised: 'player-type-hook' did not succeed previously
Applied hide-music-button
Applied hide-video-ads
Failed to apply microg-support: 'microg-support' depends on 'microg-support-bytecode-patch' but the following error was raised: 'microg-support-bytecode-patch' depends on 'spoof-player-parameters' but the following error was raised: 'spoof-player-parameters' depends on 'player-type-hook' but the following error was raised: 'player-type-hook' did not succeed previously
Applied optimize-resource
Failed to apply overlay-buttons: 'overlay-buttons' depends on 'video-id-hook' but the following error was raised: 'video-id-hook' depends on 'player-type-hook' but the following error was raised: 'player-type-hook' did not succeed previously
Failed to apply return-youtube-dislike: 'return-youtube-dislike' depends on 'video-id-hook' but the following error was raised: 'video-id-hook' depends on 'player-type-hook' but the following error was raised: 'player-type-hook' did not succeed previously
Applied settings
Failed to apply sponsorblock: 'sponsorblock' depends on 'sponsorblock-bytecode-patch' but the following error was raised: 'sponsorblock-bytecode-patch' depends on 'video-id-hook' but the following error was raised: 'video-id-hook' depends on 'player-type-hook' but the following error was raised: 'player-type-hook' did not succeed previously
Repacking patched apk
Compiling resources
Writing modified dex files
Finished!
```

### Device Environment

Samsung Galaxy S21 FE 5G, OneUI 5.1, Android 13

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.
- [X] I have written the title and contents in English.
- [X] This issue does not reproduce on unpatched YouTube or YT Music.

**Comment by inotia00** [comment]
reinstall your manager

and select YouTube v18.25.39 APK from your device

**Comment by hoang-himself** [comment]
Problem persists.

Other things I tried:
- Clear the manager's data then reinstall
- Use another device
- YT v18.24.37

**Comment by inotia00** [comment]
I don't see any solution
Try it with Revancify
It is presumed to be an issue related to ReVanced Manager

**Comment by Stale-1** [comment]
> I don't see any solution Try it with Revancify It is presumed to be an issue related to ReVanced Manager

Tried again using Revancify still the package name is still the default package name here is the log. 
https://www.mediafire.com/file/2hj70xnbqgs4edg/patch_log.txt/file


Here is the patches I selected 
https://www.mediafire.com/file/7b63qz9tzyukpu9/inotia00-patches.json/file

**Comment by S23u512** [comment]
Try to patch again using Revancify but before patching delete Patch Options file.

**Comment by Stale-1** [comment]
> Try to patch again using Revancify but before patching delete Patch Options file.

Same thing happened , it says errors due to the patch Dependencies like LithoPatchFilter etc. But when I select most of the patch (88 Default Patches) it works just fine.
I think the some patch need to be selected along with other patches for it to patch with no errors.

In the failed patch I only selected 19-22 patches but the default which is 88 patches works just fine.

**Comment by inotia00** [comment]
fixed in [revanced-patches-v2.182.1](https://github.com/inotia00/revanced-patches/releases/tag/v2.182.1)
